src/file/file.service.ts

```diff
@@ -22,9 +23,9 @@ export class FileService {
     return key;
```

Comment:
`return this.hasKey(objectID.toHexString())`

Reply:
Variable `key` is useless.

Reply:
Also remove extra line in line 15 between the comment of the function and the class' declaration.